packages/mrm-task-eslint/index.js

```diff
@@ -25,7 +48,8 @@ module.exports = function task({
 
 	// Preset
 	if (eslintPreset !== 'eslint:recommended') {
-		packages.push(`eslint-config-${eslintPreset}`);
+		const presetPackage = normalizePresetPackageName(eslintPreset);
+		packages.push(presetPackage);
```

Comment:
A variable doesn't improve readability here:

```suggestion
		packages.push(normalizePresetPackageName(eslintPreset));
```